.travis.yml

```diff
@@ -51,6 +51,7 @@ install:
 # that was created in the step above.
 - docker restart mtc_funcs
 script:
+- cd ${TRAVIS_BUILD_DIR}/admin && yarn sql-integration-test
```

Comment:
At one time the integration test rather brutally left different values in key fields like the settings table.  To be safe, we should ensure it does not alter any settings or move it to after the e2e run.